### PR TITLE
Modifications for BGP deployment

### PR DESCRIPTION
**Pull request by luis5tb**
This is to automate most of the steps to deploy in an env that is also connected to a spine-leaf setup (with 2 ToR for each compute)

### CODE REVIEW

File: devsetup/scripts/bgp-interfaces-setup.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x7'/>
+</controller>
+EOF
+
+cat << EOF > /tmp/net1.xml
+<interface type='network'>
+  <source network='default'/>
+  <model type='virtio'/>
+  <address type='pci' domain='0x0000' bus='0x06' slot='0x00' function='0x0'/>
+</interface>
```
Comment (stuggi) [inline devsetup/scripts/bgp-interfaces-setup.sh:33]: should this be the same as in https://github.com/openstack-k8s-operators/install_yamls/blob/main/devsetup/Makefile#L107 ? would it make sense to make this script generic to run by default what we currently have in the Makefile and in the BGP case also add the other?

File: devsetup/scripts/gen-edpm-compute-node-bgp.sh [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+EDPM_COMPUTE_SUFFIX=${1:-"0"}
+EDPM_COMPUTE_NAME=${EDPM_COMPUTE_NAME:-"edpm-compute-${EDPM_COMPUTE_SUFFIX}"}
+if [ ${STANDALONE} = "true" ]; then
+    EDPM_COMPUTE_VCPUS=${EDPM_COMPUTE_VCPUS:-8}
+    EDPM_COMPUTE_RAM=${EDPM_COMPUTE_RAM:-20}
+    EDPM_COMPUTE_DISK_SIZE=${EDPM_COMPUTE_DISK_SIZE:-70}
+fi
+EDPM_COMPUTE_VCPUS=${EDPM_COMPUTE_VCPUS:-8}
+EDPM_COMPUTE_RAM=${EDPM_COMPUTE_RAM:-24}
+EDPM_COMPUTE_DISK_SIZE=${EDPM_COMPUTE_DISK_SIZE:-60}
```
[the remark this answers is not in the document]
Comment (luis5tb) [reply devsetup/scripts/gen-edpm-compute-node-bgp.sh:33]: right, I'll use the defaults one here

File: devsetup/scripts/bgp-interfaces-setup.sh (hunk 1 above)
Comment (luis5tb) [reply devsetup/scripts/bgp-interfaces-setup.sh:33]: I think the problem we had was that we needed to add extra pci slots. You are suggesting to add the pci slots in the BGP case too, right?

Comment (stuggi) [reply devsetup/scripts/bgp-interfaces-setup.sh]: yes, add the pci slots in case of bgp, but not in the default one

**Review by stuggi — APPROVED** [review]
/lgtm

**Pull request merged**